src/include/ceph_fs.h

```diff
@@ -739,6 +743,7 @@ int ceph_flags_to_mode(int flags);
 				 CEPH_CAP_XATTR_SHARED)
 #define CEPH_STAT_CAP_INLINE_DATA (CEPH_CAP_FILE_SHARED | \
 				   CEPH_CAP_FILE_RD)
+#define CEPH_STAT_RSTAT        CEPH_CAP_FILE_FOO   /* for requesting rstat */
```

Comment:
I'd prefer not to  introduce new flag. how about "#define CEPH_STAT_RSTAT CEPH_CAP_GWREXTEND"

Reply:
this will be equal to this one "#define CEPH_CAP_XATTR_SHARED (CEPH_CAP_GSHARED  << CEPH_CAP_SXATTR)". How about "#define CEPH_STAT_RSTAT (CEPH_CAP_GWREXTEND << 16)"?